packages/spartan/buildinfo.json

```diff
@@ -4,8 +4,8 @@
     "spartan": {
       "kind": "git",
       "git": "https://github.com/dcos/spartan.git",
-      "ref": "0b5243b8032bf475fb2c62ec4f92142d701f35ad",
-      "ref_origin": "master"
+      "ref": "4644ab6642cf0b13fe07af53b9f04f8316aea2f0",
+      "ref_origin": "dcos/1.9"
```

Comment:
I don't see this ref in https://github.com/dcos/spartan/commits/dcos/1.9
Looks like that SHA is in https://github.com/dcos/spartan/compare/nlsun/backport-blacklist so let's use the one on the proper 1.9 branch please. https://github.com/dcos/spartan/commit/e94d5661cd271387379fd41001b80a31a1b4eab8